Makefile

```diff
@@ -66,14 +66,24 @@ reset:
 clean:
 	$(RM)
 
+lint:
+	$(CONDA_ACTIVATE)
+	$(PYTHON) -m bandit -r -f html -o bandit.html .
+	$(PYTHON) -m pylint --exit-zero cvrp test >pylint.txt
+	$(PYTHON) -m flake8 . --output-file=flake8.txt
+	$(PYTHON) -m pydocstyle . -e -s > pydocstyle.txt || exit 0
+	
+
+
 help:
```

Comment:
It doesn't work any more on Windows.
Please, fix the command "make help".
Thanks